<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>string_list_model_form</class>
 <widget class="QWidget" name="string_list_model_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>QListView</string>
   </property>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>141</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInitList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsertItem">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>插入项</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAppendItem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加项</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteCurrentItem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除当前项</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearList">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除列表</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>271</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>QPlainTextEdit</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCleartText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空文本</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowStringList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示数据模型的StringgList</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前项： row=   , column= </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
